packages/App/package.json

```diff
@@ -26,9 +32,17 @@
     "eslint": "7.14.0",
     "jest": "^26.6.3",
     "metro-react-native-babel-preset": "^0.66.2",
-    "react-test-renderer": "17.0.2"
+    "react-test-renderer": "17.0.2",
+    "get-yarn-workspaces": "1.0.2",
+    "react-native-sensitive-info" : "5.5.8",
+    "react-native-encrypted-storage": "4.0.2",
```

Comment:
`react-native-encrypted-storage` is not a development dependency. It should be moved to `dependencies`. It seems, `react-native-sensitive-info` is not used anywhere in the app, so it can be safely deleted